tests/lib/snaps/test-snapd-remodel-pc-18/grub.cfg

```diff
@@ -0,0 +1,42 @@
+set default=0
```

Comment:
Could we avoid importing copies of these files into git tree (also grubx64.efi , hooks, icons etc) and instead download/unpack them (like we do when re-packing snapd snap?) in prepare, or is it annoying to do? Main concern is they might get out of sync and we may miss actual issues.

Reply:
The snap will be published in our brand store using these files